code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -469,3 +469,83 @@ Contains:
+	resistance_flags = null
+	max_heat_protection_temperature = null
+
+/obj/item/clothing/suit/space/hardsuit/ert/sec/lp
+	armor = list("melee" = 35, "bullet" = 15, "laser" = 30, "energy" = 40, "bomb" = 10, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/lp/sec
+	name = "Loss Prevention Security Hardsuit"
+	desc = "The best of the best security staff get assigned to the ERT. Second best are given this Hardsuit as a part of the LP Team."
+	icon_state = "ert_security"
+	item_state = "ert_security"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/sec
+	armor = list("melee" = 35, "bullet" = 20, "laser" = 30,"energy" = 40, "bomb" = 10, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75)
+	hardsuit_type = "ert_security"
+	name = "Loss Prevention Security Hardsuit Helmet"
+	desc = "The helmet that comes attached to the LP Team Security Hardsuit."
+	icon_state = "hardsuit0-ert_security"
+	item_state = "hardsuit0-ert_security"
+
+/obj/item/clothing/suit/space/hardsuit/ert/lp/engi
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 100, "acid" = 75)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/lp/engi
+	name = "Loss Prevention Engineering Hardsuit"
+	desc = "The best of the best engineering staff get assigned to the ERT. Second best are given this Hardsuit as a part of the LP Team."
+	icon_state = "ert_engineer"
+	item_state = "ert_engineer"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/engi
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 100, "acid" = 75)
+	name = "Loss Prevention Engineering Hardsuit Helmet"
+	desc = "The helmet that comes attached to the LP Team Engineering Hardsuit."
+	icon_state = "hardsuit0-ert_engineer"
+	item_state = "hardsuit0-ert_engineer"
+	hardsuit_type = "ert_engineer"
+
+/obj/item/clothing/suit/space/hardsuit/ert/lp/med
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 75)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/lp/med
+	name = "Loss Prevention Medical Hardsuit"
+	desc = "The best of the best medical staff get assigned to the ERT. Second best are given this Hardsuit as a part of the LP Team."
+	icon_state = "ert_medical"
+	item_state = "ert_medical"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/med
+	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 20, "bomb" = 10, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 75)
+	name = "Loss Prevention Medical Hardsuit Helmet"
+	desc = "The helmet that comes attached to the LP Team Medical Hardsuit."
+	icon_state = "hardsuit0-ert_medical"
+	item_state = "hardsuit0-ert_medical"
+	hardsuit_type = "ert_medical"
+
+/obj/item/clothing/head/helmet/space/hardsuit/ert/lp/clown
+	name = "LP Funny Hardsuit Helmet"
```

Comment:
Uhhhhh I will double check..

Reply:
Yes this is intentional.

Reply:
If it's LP and not Loss prevention, it could stand for anything else with those letters. Laughter Protocall? I'll likely write something into the description for it later.